rccheck/src/lib.rs

```diff
@@ -1,5 +1,15 @@
 // Copyright(C) 2022, Mysten Labs
 // SPDX-License-Identifier: Apache-2.0
+
+//! THe purpose of this module is to afford the user that already does public key infrastructure on their hosts the ability to perform
```

Comment:
nit: THe

Reply:
> already does public key infrastructure

That already knows the correct public keys of other entities? (Doing PKI sounds a little strange)